runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -4771,13 +4776,23 @@ reservationLockEnter(TR::Node *node, int32_t lwOffset, TR::CodeGenerator *cg, TR
                                  node, TR::MemoryReference::createWithDisplacement(cg, objReg, lwOffset & 0x0000FFFF, lockSize), monitorReg);
       }
 
-   doneLabel->setEndInternalControlFlow();
-   generateDepLabelInstruction(cg, TR::InstOpCode::label, node, doneLabel, conditions);
+   generateLabelInstruction(cg, TR::InstOpCode::label, node, doneLabel);
+#if JAVA_SPEC_VERSION >= 19
+      /*
+       * Adjust J9VMThread ownedMonitorCount for execution paths that do not
+       * go out of line to acquire the lock. It is safe and efficient to do
+       * this unconditionally. There is no need to check for overflow.
+       */
+      generateTrg1MemInstruction(cg, TR::InstOpCode::Op_load, node, tempReg, TR::MemoryReference::createWithDisplacement(cg, metaReg, fej9->thisThreadGetOwnedMonitorCountOffset(), TR::Compiler->om.sizeofReferenceAddress()));
+      generateTrg1Src1ImmInstruction(cg, TR::InstOpCode::addi, node, tempReg, tempReg, 0x1);
```

Comment:
general comment:  you are using 0x1 or -0x1 in the code ... that just looks strange to me. is this a particular coding style you are following?

Reply:
No particular reason so I can change it.